syntax/std_seq_split_test.go

```diff
@@ -4,20 +4,21 @@ import "testing"
 
 func TestStrSplit(t *testing.T) {
 	t.Parallel()
-	AssertCodesEvalToSameValue(t,
-		`["t", "h", "i", "s", " ", "i", "s", " ", "a", " ", "t", "e", "s", "t"]`,
-		`//seq.split("","this is a test")`)
+
 	AssertCodesEvalToSameValue(t, `["this", "is", "a", "test"]`, `//seq.split(" ","this is a test") `)
 	AssertCodesEvalToSameValue(t, `["this is a test"]         `, `//seq.split(",","this is a test") `)
 	AssertCodesEvalToSameValue(t, `["th", " ", " a test"]     `, `//seq.split("is","this is a test")`)
 	assertExprPanics(t, `//seq.split(1, "this is a test")`)
+
+	AssertCodesEvalToSameValue(t,
+		`["t", "h", "i", "s", " ", "i", "s", " ", "a", " ", "t", "e", "s", "t"]`,
+		`//seq.split("","this is a test")`)
+	AssertCodesEvalToSameValue(t, `""`, `//seq.split("","") `)
```

Comment:
Strictly speaking, the expected value should be `[]`. Even though `[] =""` in arr.ai, the intent for `//seq.split` is to return an array, so it should be expressed as such.

Reply:
Fixed.